assets/src/editor/events/dates-and-times/editor-date/edit-form/edit-date-entity-form-modal.js

```diff
@@ -12,21 +12,19 @@ import { isModelEntityOfModel } from '@eventespresso/validators';
 /**
  * Internal dependencies
  */
-import EditEventDateForm from './edit-event-date-form';
-import { eventDateEntityFormSchema } from './event-date-entity-form-schema';
-import {
-	eventDateEntityFormSubmitHandler,
-} from './event-date-entity-form-submit-handler';
+import EditDateEntityForm from './edit-date-entity-form';
```

Comment:
I just went with `EditDateEntityForm` because it didn't vary too much from what it was originally: `EditEventDateForm` - I figured you wanted the 'Edit' prefix there for some reason.  Keep in mind though that this is a component, so it'd have to be `DateEntityForm`.  I'll add that to the list of things to change.